count_server/web/views/session_view.ex

```diff
@@ -0,0 +1,15 @@
+defmodule CountServer.SessionView do
+  use CountServer.Web, :view
+
+  def render("new.json", %{reply: reply}) do
+    reply
+  end
+
+  def render("create.json", %{reply: reply}) do
+    reply
+  end
+
+  def render("destroy.json", %{reply: reply}) do
+    reply
+  end
+end
```

Comment:
If you are returning 'reply' no matter the action, maybe you don't need to be pattern matching for the action.
You could substitute the three definitions for:

def render(_action, %{reply: reply}), do: reply